### PR TITLE
refactor: move footer data to side menu and remove footer

### PR DESCRIPTION
**Pull request by misiekhardcore**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

related to https://github.com/camunda/camunda/issues/23630

### CODE REVIEW

File: optimize/client/src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,31 +0,0 @@
...
-  const {optimizeVersion} = useUiConfig();
-
-  const timezoneInfo =
-    t('footer.timezone') + ' ' + Intl.DateTimeFormat().resolvedOptions().timeZone;
-
-  return (
-    <footer className="Footer">
-      <div title={timezoneInfo} className="timezone">
-        {timezoneInfo}
-      </div>
```
Comment (OmranAbazid) [inline optimize/client/src/components/Footer/Footer.tsx:24]: The timezone information is an important thing in Optimize and should not be removed.
I think it should be moved to the user menu as well. We can try adding a seperator and putting it below the copyright info.

**Review by OmranAbazid — COMMENTED** [review]
The line height is too big for us for some reason
Screenshot 2024-10-21 at 11 31 41 AM

Other webapps use a much smaller line height. See console for example:
Screenshot 2024-10-21 at 11 32 29 AM

Can you take a look and see if it is a problem in our side?

**Comment by misiekhardcore** [comment]
@OmranAbazid the line heights are different due to the carbon reset we are disabling

**Comment by OmranAbazid** [comment]
> @OmranAbazid the line heights are different due to the carbon reset we are disabling 

@misiekhardcore lets add a temporary reset in the header with a todo to remove it later

File: optimize/client/src/components/Footer/Footer.tsx (hunk 1 above)
Comment (misiekhardcore) [reply optimize/client/src/components/Footer/Footer.tsx:24]: I cant see any place where we can pass this. C3 navigation has defined props and its not really allowing to pass anything more then those props. There is `bottomElements` section, but it renders buttons and takes props to use with buttons.

**Comment by misiekhardcore** [comment]
@OmranAbazid C3 navigation is styled wirh styled-components so I can't really target that element with version info to style it. I think we need to keep it as is.
FYI Out of curiosity I enabled the carbon style reset and Optimize still looks alright, so maybe it is a good time to consider enabling that reset for good.

File: optimize/client/src/components/Footer/Footer.tsx (hunk 1 above)
Comment (misiekhardcore) [reply optimize/client/src/components/Footer/Footer.tsx:24]: I will go for it and use that `bottomElements` for this, but it is really hacky

**Comment by OmranAbazid** [comment]
> @OmranAbazid C3 navigation is styled wirh styled-components so I can't really target that element with version info to style it. I think we need to keep it as is. FYI Out of curiosity I enabled the carbon style reset and Optimize still looks alright, so maybe it is a good time to consider enabling that reset for good.

We will have issue with p and bold tags across the application. We have an issue to fix those and enable the css reset.
Let's keep this issue on hold until we fix that css reset. I will try to do it as soon as possible and afterwards, i will look into merging this.

File: optimize/client/src/components/Footer/Footer.tsx (hunk 1 above)
Comment (OmranAbazid) [reply optimize/client/src/components/Footer/Footer.tsx:24]: @misiekhardcore can we try the suggested solution in the slack thread above?

**Comment by misiekhardcore** [comment]
@OmranAbazid I've seen [this convo](https://camunda.slack.com/archives/C049YH1C202/p1729768509111639) and I moved the timezone info to that section

**Review by OmranAbazid — APPROVED** [review]

**Pull request merged**